# Service to Set the tracking velocity and start tracking. 
# Velocity vector in mm/s
float64 vx  
float64 vy  
float64 vz  
---
int64 ret
string msg
